client/encointer-api-client-extension/src/extrinsic_params.rs

```diff
@@ -14,11 +14,11 @@ pub type CommunityCurrencyTipExtrinsicParams<T> = GenericExtrinsicParams<T, Comm
 pub type CommunityCurrencyTipExtrinsicParamsBuilder =
 	GenericAdditionalParams<CommunityCurrencyTip, Hash>;
 
-pub type EncointerXt<Call> = UncheckedExtrinsicV4<
+pub type EncointerXt<Call> = UncheckedExtrinsic<
 	ExtrinsicAddress,
 	Call,
 	Signature,
-	GenericSignedExtra<CommunityCurrencyTip, Nonce>,
+	GenericTxExtension<CommunityCurrencyTip, Nonce>,
```

Comment:
With this update, we implicitly introduce the UncheckedExtrinsicV5. However, it is entirely backwards compatible, and should not introduce any errors. (The parachain will support the V5 too with the stable2412 update).